README.md

```diff
@@ -1,2 +1,23 @@
-# student-developer-setup
-Bootstrapped instructions for setting up development environments for student developers
+# Yan Zhuang student-developer-setup
+
+1) Created fork on GitHub
+
+2) Created README.md
+
+3) Pushed to GitHub, created pull request
+
+4) Using Pycharm (Working on Windows System)
+
+5) Created `hello.py` as a "hello world" program; can be built via the `build.sh` script.
+
+6-7) Created requirement.txt and add pre-commit as required package
+
+8) Created pre-commit yaml configuration file using its sample official config file
+
+9) Installed docker
+
+10) Read the guide for how to use docker
+
+13) Created a Dockerfile Zhuang folder that builds and runs the "hello world" program
```

Comment:
From Issue #5, 

> Include this Dockerfile and build instructions or script in your PR.

So either in the README or in a build script you should have steps to build the Docker image and run the image as a container.

Reply:
Fixed it